data/db/src/commonMain/sqldelight/io/github/droidkaigi/feeder/data/FeedItemPodcast.sq

```diff
@@ -9,7 +9,8 @@ CREATE TABLE feedItemPodcast(
     enTitle Text NOT NULL,
     jaSummary Text NOT NULL,
     enSummary Text NOT NULL,
-    link Text NOT NULL
+    link Text NOT NULL,
+    podcastLink Text NOT NULL
```

Comment:
Almost LGTM!
It seems that the DB schema has changed. Can users of existing apps launch the app? 🙋

Reply:
Thanks for the review!
I just checked, but it seems that an error occurs when updating from an existing version. Should I add a migration process?

Reply:
Thank you for checking!
Ideally, It should be able to erase the current data and start it when migration is required.
In Room, it seems that it could be realized by fallbackToDestructiveMigration.
I'll look into it a little. 👀 
https://stackoverflow.com/questions/65200099/how-to-see-update-database-version-is-there-something-like-fallbacktodestructi

Reply:
Thank you for the reference article!
I agree! I'll also investigate to achieve a feature similar to fallbackToDestructiveMigration in Sqldelight. 👀

Reply:
👀 

```
2021-03-22 09:48:29.511 14794-14794/io.github.droidkaigi.feeder.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.github.droidkaigi.feeder.debug, PID: 14794
    android.database.sqlite.SQLiteException: near "IF": syntax error (code 1 SQLITE_ERROR): , while compiling: DROP TABLE feedItemPodcast IF EXISTS TABLE_NAME
        at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
        at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1045)
        at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:652)
        at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
        at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:61)
        at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:33)
        at android.database.sqlite.SQLiteDatabase.compileStatement(SQLiteDatabase.java:1223)
        at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.compileStatement(FrameworkSQLiteDatabase.java:64)
        at com.squareup.sqldelight.android.AndroidSqliteDriver$execute$1.invoke(AndroidSqliteDriver.kt:136)
        at com.squareup.sqldelight.android.AndroidSqliteDriver$execute$1.invoke(AndroidSqliteDriver.kt:19)
        at com.squareup.sqldelight.android.AndroidSqliteDriver.execute(AndroidSqliteDriver.kt:117)
        at com.squareup.sqldelight.android.AndroidSqliteDriver.execute(AndroidSqliteDriver.kt:136)
        at com.squareup.sqldelight.db.SqlDriver$DefaultImpls.execute$default(SqlDriver.kt:56)
        at io.github.droidkaigi.feeder.data.db.DatabaseImpl$Schema.migrate(DatabaseImpl.kt:115)
```

Reply:
Probably the following is correct as a SQL statement. However, I am afraid that this will not recreate the table. 👀 
`DROP TABLE IF EXISTS feedItemPodcast;`

Reply:
I missed a SQL syntax error. Thank you for pointing out!
I made a change to create a table at the timing of migration 👀